core/src/main/scripts/importer/validateData.py

```diff
@@ -313,9 +317,14 @@ def _validate_file(self):
 
             # parse start-of-file comment lines, if any
             if not self.processTopLines(top_comments):
-                self.logger.error(
-                    'Invalid header comments, file cannot be parsed')
-                return
+                if not self.relaxed_mode:
+                    self.logger.error(
+                        'Invalid header comments, file cannot be parsed')
+                    return
+                else:   
+                    self.logger.info('Ignoring missing or invalid header comments. '
+                        'Continuing with validation...')                    
```

Comment:
And if we do want to disable the fail-fast behaviour and continue validating other files even if no samples are defined, as @n1zea144 asks, that’s the following block in the `validate_study` function:

```Python
    if defined_sample_ids is None:
        logger.error("Sample file could not be parsed. Please fix "
                     "the problems found there first before continuing.")
        return
```